docs/socket-groups.md

```diff
@@ -56,14 +56,19 @@ Every next link in this group gives then another 100% overhead.
 ## 2. Backup
 
 This solution is more complicated and more challenging for the settings,
```

Comment:
```suggestion
The settings for this grout type are more complicated and more challenging.
```

Reply:
What I actually meant is that it may be challenging to define proper settings by the user in accordance to network conditions and desired latency, and "complicated" is the overall solution of the backup groups. I rephrased it a little bit differently.

Reply:
"The configuration of Backup groups is somewhat more complicated than with the other group types. In particular, it may be challenging to arrive at the optimal settings for a a given set of network conditions and desired latency."